heltour/tournament/tasks.py

```diff
@@ -197,11 +197,16 @@ def rating(p):
         # Get the rating AFTER the game
         p = pairings_by_date_lt[-1][1]
         if p.game_id() is not None:
-            game_meta = lichessapi.get_game_meta(p.game_id(), priority=0, timeout=300)
-            player_meta = game_meta['players']['white'] if p.white == player else \
-                game_meta['players']['black']
-            if 'ratingDiff' in player_meta:
-                return player_meta['rating'] + player_meta['ratingDiff']
+            try:
+                game_meta = lichessapi.get_game_meta(p.game_id(), priority=0, timeout=300)
+            except Exception as e:
```

Comment:
bare excepts like this are pretty dangerous, I'd prefer if we caught `ApiClientError` specifically. 

If we want to catch everything, then we should also log the full stacktrace.  I'm in favour of logging it in both cases. this can be done with with the `traceback` module:
```python
import traceback
stacktrace = traceback.format_exception(e)
logger.error(stacktrace)
```

Reply:
I tried to do this below, though I had to use a different syntax for format_exception because we're using an older python version.

I will also add that I don't know if catching everything is actually helpful, I don't even know how to trigger anything but an ApiClientError.